### DISCUSSION TITLE
DDS Security support

### DISCUSSION THREAD

**Discussion by feiertagk** [jmmorato/openddsharp#174] [General]
I need to use the DDS Security feature for authentication and encryption, descripted here: 
https://opendds.readthedocs.io/en/latest/devguide/dds_security.html

**Comment by jmmorato** [comment]
DDS Security is not supported in OpenDDSharp yet, and I don't plan to work on it soon (read [Roadmap Overview](https://github.com/jmmorato/openddsharp#roadmap-overview)).

However, there are other ways to make it done sooner:
 - [Collaborate](https://github.com/jmmorato/openddsharp#implement-it-yourself) with your code
   - I will be glad to review your code and guide you during the implementation
- [Sponsor](https://github.com/jmmorato/openddsharp#sponsor-it) the feature implementation
